Casks/y/yubihsm2-sdk.rb

```diff
@@ -12,7 +12,7 @@
 
   livecheck do
     url "https://developers.yubico.com/YubiHSM2/Releases/"
-    regex(%r{href=.*?/yubihsm2-sdk[._-]v?(\d+(?:[.-]\d+)+)-darwin-#{arch}\.pkg}i)
+    regex(%r{href=.*?/yubihsm2-sdk[._-]v?(\d+(?:[.-](\d|\w)+)+)-darwin-#{arch}\.pkg}i)
```

Comment:
```suggestion
    regex(%r{href=.*?/yubihsm2-sdk[._-]v?(\d+(?:[.-]\d+)+[a-z]?)-darwin-#{arch}\.pkg}i)
```

This should be more explicit, as we are only looking for an optional character at the end of the version string.